purview/models/product.py

```diff
@@ -0,0 +1,18 @@
+# SPDX-License-Identifier: GPL-3.0+
+
+from __future__ import unicode_literals
+
+from neomodel import UniqueIdProperty, RelationshipFrom, StringProperty
+
+from purview.models.base import PurviewStructuredNode
+
+
+class Product(PurviewStructuredNode):
+    """Definition of a released Product"""
+
+    attached_koji_tag = RelationshipFrom('.koji.KojiTag', 'ATTACHED_TO')
```

Comment:
Attached aren't the right terms. How about `related` or `associated`? This should also be plural.